server/models/group.js

```diff
@@ -0,0 +1,19 @@
+import mongoose from 'mongoose';
+
+/**
+ * Group Schema
+ *
+ * id - default id given by mongodb
+ * users - list of user objects that are included in the group
+ * escalationPolicy - escalation policy for this group
+ */
+
+const GroupSchema = new mongoose.Schema({
+  users: [mongoose.Schema.Types.ObjectId],
+  escalationPolicy: {
+    type: mongoose.Schema.ObjectId,
```

Comment:
This doesn't actually embed the document, it just adds the `ObjectId` that can get the document once you call the `.populate('escalationPolicy')` method when you're getting the group object.